darts/models/ensemble_model.py

```diff
@@ -23,50 +24,114 @@ class EnsembleModel(ForecastingModel):
     models
         List of forecasting models whose predictions to ensemble
     """
-    def __init__(self, models: List[ForecastingModel]):
+    def __init__(self, models: Union[List[ForecastingModel], List[GlobalForecastingModel]]):
         raise_if_not(isinstance(models, list) and models,
                      "Cannot instantiate EnsembleModel with an empty list of models",
                      logger)
-        raise_if_not(all(isinstance(model, ForecastingModel) for model in models),
-                     "All models must be instances of darts.models.ForecastingModel",
+
+        is_local_ensemble = all(isinstance(model, ForecastingModel) and not isinstance(model, GlobalForecastingModel)
```

Comment:
Yeah, but it covers a test case that is checking whether there are classes provided instead of objects, see `test_input_models_local_models()` in `test_ensemble_models.py`. Ok?